pkgs/development/python-modules/cppyy-cling/default.nix

```diff
@@ -0,0 +1,80 @@
+{
+  lib,
+  fetchFromGitHub,
+  fetchurl,
+  zlib,
+  cmake,
+  pkg-config,
+  setuptools,
+  darwin,
+  buildPythonPackage,
+  clang-tools,
+  stdenv,
+  rootVersion ? "6.32.08",
+}:
+buildPythonPackage rec {
+  pname = "cppyy-cling";
+  version = "6.32.8";
+
+  sourceRoot = ".";
```

Comment:
probably not necessary

Reply:
Simplest way to make it build, given circumstances / structure of the underlying projects.

Reply:
Without the line, it does break since there are srcs instead of a single src.